tools/depends/target/pythonmodule-pil/pillow-crosscompile.patch

```diff
@@ -3,7 +3,7 @@
 @@ -156,6 +156,16 @@
  
          _add_directory(include_dirs, "libImaging")
- 
+
```

Comment:
you introduced the whitspace changes at 4f7fc770431f58a97c7f492b146679ea563ce19d for this file.
drop them
I wonder why this even does apply